impl/glfw/glfw.odin

```diff
@@ -133,11 +133,13 @@ update_mouse :: proc() {
 }
 
 update_display_size :: proc() {
-    io := imgui.get_io();
     w, h := glfw.get_window_size(state.window);
-    display_w, display_h := glfw.get_framebuffer_size(state.window);
-    io.display_size = { f32(w), f32(h) };
-    if w > 0 && h > 0 do io.display_framebuffer_scale = { f32(display_w) / f32(w), f32(display_h) / f32(h) };
+    if w > 0 && h > 0 {
+        io := imgui.get_io();
+        display_w, display_h := glfw.get_framebuffer_size(state.window);
+        io.display_size = { f32(w), f32(h) };
```

Comment:
io.display_size shold still be set even if `w` & `h` are not above 0 (otherwise dear imgui doesn't know the display is so small